Assets/Scripts/ThrowableObjects.cs

```diff
@@ -4,16 +4,66 @@
 
 public class ThrowableObjects : InteractableObjects
 {
+    // the objects needs to be iteractable by clicking a key, being picked up  V
+    // when he is picked up cannot be iteractble by other players
+    // needs to have weight parameter for physics - using rigidbody for mass calc  V
+    // speed parameter of moving changed by player stats V
+    // needs to have a direction of throw V
 
-    // Start is called before the first frame update
-    void Start()
+    float ThrowForce = 600f; 
+    Vector3 objectPos;
+    float Distance; 
+
+    public bool canPick = true; 
```

Comment:
global public variables should start with large letter (CanPick), or make it private